test/test_prototype_transforms.py

```diff
@@ -1665,205 +1665,3 @@ def test__transform(self):
         assert isinstance(ohe_labels, features.OneHotLabel)
         assert ohe_labels.shape == (4, 3)
         assert ohe_labels.categories == labels.categories == categories
-
-
-class TestAPIConsistency:
-    @pytest.mark.parametrize("antialias", [True, False])
-    @pytest.mark.parametrize(
-        "inpt",
-        [
-            torch.randint(0, 256, size=(1, 3, 256, 256), dtype=torch.uint8),
-            PIL.Image.new("RGB", (256, 256), 123),
-            features.Image(torch.randint(0, 256, size=(1, 3, 256, 256), dtype=torch.uint8)),
-        ],
-    )
-    def test_random_resized_crop(self, antialias, inpt):
```

Comment:
I've removed this while moving this class, since this was already present in the automatic consistency checks.